src/main/java/org/folio/consortia/service/TenantService.java

```diff
@@ -0,0 +1,16 @@
+package org.folio.consortia.service;
+
+import org.folio.pv.domain.dto.TenantCollection;
+
+public interface TenantService {
+
+  /**
+   * Gets tenant collection by search query.
+   *
+   * @param query the query
+   * @param offset the offset
+   * @param limit the limit
+   * @return tenant collection
+   */
+  TenantCollection get(String query, Integer offset, Integer limit);
```

Comment:
Lets do out contract more specific and simple, I think we don't need to support cql, at least for now

Reply:
fixed